packages/react-server-integration-tests/package.json

```diff
@@ -13,6 +13,7 @@
     "react-dom": "~0.14.2",
     "react-server": "../react-server",
     "react-server-cli": "../react-server-cli",
+    "react-server-module-tagger": "../react-server-module-tagger",
```

Comment:
I make this transitive dependency a direct dependency so that `react-server-module-tagger`'s `prepublish` script is called; otherwise, we get module not found errors from `react-server-gulp-module-tagger` and `babel-plugin-react-server`